docs/Examples.md

```diff
@@ -0,0 +1,107 @@
+In the following sections, we present example configurations for a variety of Git Providers and Issue Trackers combinations.
+
+## Bitbucket & Jira
+
+```yaml
+apiVersion: charts.pelorus.konveyor.io/v1alpha1
+kind: Pelorus
+metadata:
+  name: bitbucket-jira-example-configuration
+spec:
+  exporters:
+    global: {} # this line is necessary?
+    instances:
+      - app_name: deploytime-exporter
+        exporter_type: deploytime
+        extraEnv:
+          - name: NAMESPACES
+            value: <namespace1,namespace2>
+      - app_name: committime-exporter
+        exporter_type: committime
+        extraEnv:
+          - name: GIT_PROVIDER
+            value: bitbucket
+          - name: NAMESPACES
+            value: <namespace1,namespace2>
+          - name: API_USER
+            value: <bitbucket_user>
+          - name: TOKEN
+            value: <bitbucket_token>
+      - app_name: failure-exporter
+        exporter_type: failure
+        extraEnv:
+          - name: PROVIDER
+            value: jira
+          - name: API_USER
+            value: <jira_username>
+          - name: TOKEN
+            value: <jira_token>
+          - name: SERVER
+            value: <url>
+          - name: PROJECTS
+            value: <project1,project2>
+```
+
+## GitHub
+
+In this example, we use GitHub both as the Git provider and the Issue tracker.
+
+> **NOTE:** If your GitHub repository is public, there no is no need to set API_USER and TOKEN for Commit time Exporter.
+
+```yaml
+apiVersion: charts.pelorus.konveyor.io/v1alpha1
+kind: Pelorus
+metadata:
+  name: github-example-configuration
+spec:
+  exporters:
+    global: {} # this line is necessary?
+    instances:
+      - app_name: deploytime-exporter
+        exporter_type: deploytime
+        extraEnv:
+          - name: NAMESPACES
+            value: <namespace1,namespace2>
+      - app_name: committime-exporter
+        exporter_type: committime
+        extraEnv:
+          - name: NAMESPACES
+            value: <namespace1,namespace2>
+          - name: API_USER
+            value: <github_user>
+          - name: TOKEN
+            value: <github_token>
+      - app_name: failure-exporter
+      - app_name: failure-exporter
+        exporter_type: failure
+        extraEnv:
+          - name: PROVIDER
+            value: github
+          - name: PROJECTS # works for github and service now????
+            value: <project1,project2>
+          - name: API_USER
+            value: <github_user>
+          - name: TOKEN
+            value: <github_token>
+```
+
+## Jira
```

Comment:
Here was an idea

Instead of writing multiple combinations, why don't we write the configuration to each one separate, and the user gathers the 2 he wants together  ?

Reply:
Talked to Wes about it. It is not the focus of these examples, so we will delete it